.vsts.release.yml

```diff
@@ -295,3 +318,29 @@ stages:
         }
         Invoke-RestMethod @assetsParams
       displayName: Create agent release on Github
+
+- stage: CreatePRs
+  dependsOn: Release
+  jobs:
+  ################################################################################
+  - job: create_ado_prs
+  ################################################################################
+    displayName: Create PRs in AzureDevOps
+    pool:
+      vmImage: ubuntu-16.04
+
+    steps:
+    - checkout: self
+
+    - bash: |
+        set -x
+        git checkout releases/$(version)
+        cd release
+        npm install
+        ls
+        node createAdoPrs.js $(version)
+      displayName: Create PRs in AzureDevOps and AzureDevOps.ConfigChange
+      condition: ne(variables['version'], '')
+      env:
+        USER: 'damccorm'
```

Comment:
can we use a service account or make this a variable along side the AdoPAT?

Reply:
I want to move it to a service account, but wanted to get it up and running first before dealing with that. Agreed on making it a variable though - updated